drivers/platform/stm32/stm32_spi.c

```diff
@@ -714,5 +747,6 @@ const struct no_os_spi_platform_ops stm32_spi_ops = {
 	.remove = &stm32_spi_remove,
 	.transfer = &stm32_spi_transfer,
 	.dma_transfer_async = &stm32_spi_dma_transfer_async,
-	.dma_transfer_sync = &stm32_spi_dma_transfer_sync
+	.dma_transfer_sync = &stm32_spi_dma_transfer_sync,
```

Comment:
Please make a separate commit where you rename this line (and all the implications thereof) to:
`.transfer_dma_sync = &stm32_spi_transfer_dma_sync,`

This will break some projects, but you can fix them in the same commit. I want to keep the naming consistent like:

```
stm32_spi_transfer_dma
^     ^       ^    ^--- method (could also be interrupt based or other)
|     |       |
|     |       |--- action
|     |      
|     |--- interface
|
|--- platform (or top level no_os)

Reply:
it's semantics but i think it's important